internal/cmd/flink/command_compute_pool_use.go

```diff
@@ -0,0 +1,38 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
+)
+
+func (c *command) newComputePoolUseCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "use <id>",
+		Short:             "Use a Flink compute pool in subsequent commands.",
+		Long:              "Choose a Flink compute pool to be used in subsequent commands which support passing a compute pool with the `--compute-pool` flag.",
+		Args:              cobra.ExactArgs(1),
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validComputePoolArgs),
+		RunE:              c.computePoolUse,
+	}
+
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *command) computePoolUse(cmd *cobra.Command, args []string) error {
+	if err := c.Context.SetCurrentFlinkComputePool(args[0]); err != nil {
```

Comment:
This should call `DescribeFlinkComputePool` to prevent the user from successfully running `use` on an invalid compute pool. Current use commands all appear to do some form of validation.

Reply:
Wrote a note to do this, then forgot. Thanks 🤦‍♂️